test/langtools/tools/javac/lambda/8258897/CaptureVariables.java

```diff
@@ -0,0 +1,65 @@
+import java.util.function.Supplier;
+
+/**
```

Comment:
It would be nice to tweak the test to so that:
-the comment with `@test` is first, and imports follow
-`@test` is augmented with `/nodynamiccopyright/`
-`@bug` is added
-`@summary` is added

(applies to both tests)

Reply:
And the action tag `@run main CaptureVariables` should be added.

@hltj Some documents are helpful to you: 
[Regression Test Harness for the JDK: jtreg](http://openjdk.java.net/jtreg/index.html)
[The JDK Test Framework: Tag Language Specification](http://openjdk.java.net/jtreg/tag-spec.html)

Reply:
Thanks @lahodaj and @lgxbslgx ! I will add the tags.

Reply:
@lahodaj I think the tests in this case can use the copyright header instead of `/nodynamiccopyright/`. What's your opinion?

Reply:
@lgxbslgx - you are right, I am sorry. `/nodymiccopyright/` is used when line numbers are significant, and I somehow thought they are here, but you are right they are not. So there should be a normal header here.

Reply:
I added the copyright comments moments ago. Thanks.